Esri/arcgis-maps-sdk-swift-samples
# [New] Style features with custom dictionary

## Description

This PR implements `Style features with custom dictionary` in `Visualization` category.
URL to README: [ReadME](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-StyleFeaturesWithCustomDictionary/Shared/Samples/Style%20features%20with%20custom%20dictionary)

## Linked Issue(s)

- `swift/issues/4074`

## How To Test

- Toggle between "Style File" and "Web Style" to see the dictionary symbols on the map change.

## Screenshots

![style-features-with-custom-dictionary screenshot](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/53d0e0d8-9c2c-4117-9d43-2dc2f7d877a5)
